CHANGELOG.md

```diff
@@ -77,14 +77,14 @@ To release a new version, please update the changelog as followed:
   - `test_utils_predict.py` added to reproduce and fix issue #288 (by @2wins).
 
 ### Changed
-- Util:
-  - `predict` function modified to fix #565 (by @2wins).
 
 ### Deprecated
 
 ### Removed
 
 ### Fixed
+- Util:
+  - `predict` function modified to fix #565 (by @2wins).
```

Comment:
You did not follow my comment:
````
- Issue #288 related to `tl.utils.predict` Fixed - predict np.vstack problem when batch_size if no factor of test dataset size
```
This is a lot more precise.

Reply:
@DEKHTIARJonathan Sorry, I'll follow you.

Reply:
I think you can just copy+paste my example, it is way enough. We do not have hundreds of bug fixes per release (hopefully).

Reply:
I read your mention and I think it should be modified as follows:
- Issue #565 related to `tl.utils.predict` fixed
  - predict `np.hstack` problem in which the results for multiple batches are stacked along `dim=1`

Reply:
Why would you create a sublist for only one point ?
I would absolutely agree with if there are many points that you would like to mention is it the case?

I mean something like this:

``` 
- Issue #565 related to `tl.utils.predict` fixed
  - predict `np.hstack` problem in which the results for multiple batches are stacked along `dim=1`
  - reason 2
  - ...
```

Reply:
OK, it should be 
- Issue #565 related to `tl.utils.predict` fixed - predict `np.hstack` problem in which the results for multiple batches are stacked along `dim=1`